.circleci/config.yml

```diff
@@ -524,7 +524,7 @@ jobs:
           command: |
             set -e
             cd packages/celotool
-            ./ci_test_transfers.sh checkout master
+            ./ci_test_transfers.sh checkout asaj/key-rotation-plus-enode
```

Comment:
do you intend to keep these pointing to your branch? Do you fix set it back in a follow-up PR once tests pass?

Reply:
Yeah, you need to do it this way in order to run the end-to-end tests properly